Respositories/EventRespository.cs

```diff
@@ -0,0 +1,58 @@
+﻿using GatesApp.Models;
+
+namespace GatesApp.Respositories
+{
+    public class EventRespository
+    {
+        private List<Event> events = new List<Event>();
+
+        public void EventRepository()
+        {
+            events.Add(new Event("Lunch Break", new TimeSpan(01, 00, 00), 1, 1));
+            events.Add(new Event("Smoke Break", new TimeSpan(00, 15, 00), 1, 2));
+            events.Add(new Event("Toilet Break", new TimeSpan(00, 10, 00), 1, 3));
+        }
+
+        public List<Event> GetEvent() => events;
+
+        public Event GetEvent(string oneEvent)
+        {
+            var currentEvent = events.FirstOrDefault(x => x.NameOfEvent == oneEvent);
+
+            return currentEvent;
+        }
+        public int LunchAmmount()
+        {
+            var random = new Random();
```

Comment:
Taip ir likau nesupratusi, kas visi sitie amounts yra. Pagal pavadinima tai pertrauku kiekiai (tarkim zmogus turejo dvi rukymo pertraukas), bet kitoje programos dalyje matau, kad tu i tai ziuri kaip i laika, kuri atimineji is bendro darbo laiko. Netgi issiaiskinus, kad tai laikas lieka neaisku, koks tai laiko vienetas. Taigi variable pavadinimas yra klaidinantis. Ir siaip, laiko intervalo nereikejo kietai ikoduoti - ta reikejo suprogramuoti fiksuojant atejimo/isejimo laikus ir pagal juos paskaiciuojant intervala.